ironfish/src/account/accounts.test.ts

```diff
@@ -44,4 +50,78 @@ describe('Accounts', () => {
     expect(node.accounts['headHash']).toEqual(blockB3.header.hash.toString('hex'))
     expect(getTransactionsSpy).toBeCalledTimes(8)
   }, 8000)
+
+  it.only('should reproduce issue', async () => {
```

Comment:
can we make this comment a bit more descriptive? e.g. what issue?

Reply:
changed the test name and removed the only